frontend/indigenousapprenticeship/src/js/components/Router.tsx

```diff
@@ -29,10 +29,10 @@ export const Router: React.FC = () => {
   const menuItems = [
     <MenuLink
       key="search"
-      href={indigenousApprenticeshipPaths.helloWorld()}
+      href={indigenousApprenticeshipPaths.home()}
       text={intl.formatMessage({
-        defaultMessage: "Hello World",
-        description: "Label displayed on the Hello World menu item.",
+        defaultMessage: "Home",
+        description: "Label displayed on the IAP menu item.",
```

Comment:
Translators probably won't know what *IAP* is :smile:.  Can we use a more descriptive description?